keyboards/durgod/k320/keymaps/default_iso/keymap.c

```diff
@@ -0,0 +1,96 @@
+      KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,            KC_RSFT,  KC_ENT,   KC_UP,
+      KC_LCTL,  KC_LGUI,  KC_LALT,                                KC_SPC,                                 KC_RALT,  MO(_FUNC),KC_APP,   KC_RCTL,  KC_LEFT,  KC_DOWN,
+                                                                                                                                                            KC_RGHT
+  ),
+  /* Keymap _FUNC: Function Layer
+   * ,-----------------------------------------------------------.  ,--------------.
+   * |   |Play|Stop|Prev|Next| |Mute|Vol+|Vol-|    | |  |  |  |  |  |    |    |    |
+   * |-----------------------------------------------------------|  |--------------|
+   * |   |   |   |   |   |   |   |   |   |   |   |   |   |       |  |    |    |    |
+   * |-----------------------------------------------------------|  |--------------|
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |  |    |    |    |
+   * |-------------------------------------------------------    |  `--------------'
+   * |       |   |   |   |   |   |   |   |   |   |   |   |  |    |
+   * |-----------------------------------------------------------|        ,----.
+   * |     |  |   |   |   |   |   |   |   |   |   |   |          |        |    |
+   * |-----------------------------------------------------------|   ,-------------.
+   * |    |Lock|    |                       |    |Func |    |    |   |   |    |    |
+   * `-----------------------------------------------------------'   `-------------'
+   */
+  [_FUNC] = LAYOUT_tkl_iso( /* Function Layer */
+      _______,  KC_MPLY,  KC_MSTP,  KC_MRWD,  KC_MFFD,  KC_MUTE,  KC_VOLD,  KC_VOLU,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+      _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,
+      _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,
+      _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+      _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,  _______,
+      _______,  KC_TGUI,  _______,                                _______,                                _______,  _______,  _______,  _______,  _______,  _______,
+                                                                                                                                                            _______
+  )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```

Comment:
Actually, you could do this with bootmagic. 

https://docs.qmk.fm/#/feature_bootmagic?id=keycodes

And you can use `keymap_config.no_gui` to check the status.

Reply:
Hi Drashna,

I just copied this functionality from the pre-existing `default` keymap - _personally_ I have no use for this functionality so I'll happy delete it from this map if you feel strongly about it, but I assumed that consistency across at least the `default` maps was a good thing so I left it in.

Please let me know what you'd like me to do, and I'll get on with it :)

Reply:
That should be fine, then.

Reply:
Thanks - added a commit to remove the winkey lock functionality in `default_iso`.